### DISCUSSION TITLE
No body or Footer

### DISCUSSION THREAD

**Discussion by aidanrmwatt** [bootscore/bootscore#261] [General]
Have just wiped WP and done a new WP installation on hostserver followed by new install of Woocommerce, then Bootscore then Bootscore woocommerce plug in. On loading page I get nothing!! The body tags are empty.
![image](https://user-images.githubusercontent.com/27211079/190516671-e5f84172-bd32-46b9-baee-2bc2d9864090.png)
I have copied default page - from Bootscore to bootscorewoocommerce and changed name to page. Other themes appear to be working.
What have I done wrong.

**Comment by crftwrk** [comment]
This is really hard to check without having a link to your site. I have some questions:

- Did you finished the WooCommerce setup?
- Does this affect only Woo pages and normal pages/posts work fine or is this on all content?
- Do you have installed and enabled https://github.com/bootscore/bootcommerce-child?

**Comment by aidanrmwatt** [comment]
Have taken off the block so site is visible.

1. Completely finished woocommerce set up all 5 stages
2. affects all pages
3. Yes I do - Bootscore as well but not active

I did delete both and re-install when issue first arose.
The download came with following files and folders
![image](https://user-images.githubusercontent.com/27211079/190664822-bd9565d9-678d-463d-aec2-80adfd0012b1.png)
Which mirrors what I Have on the server
![image](https://user-images.githubusercontent.com/27211079/190665618-eb5f2064-1a21-4cae-8fdc-335edc1d001c.png)

I installed just one other plug in - all in one security but only turned on maintenance.

Will do a complete re-install this evening o no rush with the answer.

**Comment by aidanrmwatt** [comment]
Is bootscore having issues after the latest updates on WP or woocommerce. getting error log of "end of script output before headers" and page content is not appearing.
www.blacksmithhouse.de

Aidan

Tel +49 162 607 9496Email ***@***.*** 

    On Friday, 16 September 2022 at 12:22:12 CEST, Basti ***@***.***> wrote:  
 
 


This is really hard to check without having a link to your site. I have some questions:
   
   - Did you finished the WooCommerce setup?
   - Does this affect only Woo pages and normal pages/posts work fine or is this on all content?
   - Do you have installed and enabled https://github.com/bootscore/bootcommerce-child?

I cannot see if your screenshot is from the editor or the dev console. If it's a screenshot from your editor, then there is a <title> tag which should be not there because title is done by WordPress. And of course the <?php wp_head(); ?> is missing https://github.com/bootscore/bootcommerce-child/blob/main/header.php (line 31). Without wp_head() nothing works.

**Comment by crftwrk** [comment]
> Is bootscore having issues after the latest updates on WP or woocommerce. 

You are using v5.2.1.1. Please update to v.5.2.2.1 https://github.com/bootscore/bootscore/releases/download/v5.2.2.1/bootscore-main.zip. This update fixes outdated templates coming with WooCommerce 7.0.1. There are no breaking changes, simply use the theme-uploader to update.

> getting error log of "end of script output before headers" and page content is not appearing.

There are no errors on your site. Checking other pages like https://blacksmithhouse.de/re-starting-the-blog/, content is displayed well? But back to your first question "No body or Footer", the header has `display: none;` on the frontpage:

```css
.page-id-203 .fixed-top {
  display: none !important;
}
```

Maybe there are more CSS rules addressing the frontpage?

**Comment by aidanrmwatt** [comment]
Well that didn't work - update done and now nothing the white screen of aaaaggghhhh

Aidan

Tel +49 162 607 9496Email ***@***.*** 

    On Monday, 7 November 2022 at 11:10:58 CET, Basti ***@***.***> wrote:  
 
 



Is bootscore having issues after the latest updates on WP or woocommerce.


You are using v5.2.1.1. Please update to v.5.2.2.1 https://github.com/bootscore/bootscore/releases/download/v5.2.2.1/bootscore-main.zip. This update fixes outdated templates coming with WooCommerce 7.0.1. There are no breaking changes, simply use the theme-uploader to update.


getting error log of "end of script output before headers" and page content is not appearing.


There are no errors on your site. Checking other pages like https://blacksmithhouse.de/re-starting-the-blog/, content is displayed well? But back to your first question "No body or Footer", the header has display: none; on the frontpage:
.page-id-203 .fixed-top {
  display: none !important;
}
Maybe there are more CSS rules addressing the frontpage?

**Comment by crftwrk** [comment]
You can downgrade to earlier version any time https://github.com/bootscore/bootscore/releases/download/v5.2.1.1/bootscore-main.zip. If you want, you can create an admin account and send login data to store@bootscore.me. Then I can quick check what's wrong on your site.

**Comment by aidanrmwatt** [comment]
I am getting lots of internal server errors which I am unable to find the source of.

Aidan

Tel +49 162 607 9496Email ***@***.*** 

    On Monday, 7 November 2022 at 13:13:31 CET, Basti ***@***.***> wrote:  
 
 


You can downgrade to earlier version any time https://github.com/bootscore/bootscore/releases/download/v5.2.1.1/bootscore-main.zip. If you want, you can create an admin account and send login data to ***@***.*** Then I can quick check what's wrong on your site.

**Reply by crftwrk** [reply]
> I am getting lots of internal server errors which I am unable to find the source of.

This has for sure nothing to do with the theme. Try to install a copy to another server.

**Comment by aidanrmwatt** [comment]
Just changed theme to 2023 and whilst everything is all over the place the content has reappeared.


Aidan

Tel +49 162 607 9496Email ***@***.*** 

    On Monday, 7 November 2022 at 14:09:33 CET, Basti ***@***.***> wrote:  
 
 



I am getting lots of internal server errors which I am unable to find the source of.


This has for sure nothing to do with the theme. Try to install a copy to another server.

**Comment by aidanrmwatt** [comment]
activated WP de bug mode and have this report
Deprecated: Return type of Requests_Cookie_Jar::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php on line 73

Deprecated: Return type of Requests_Cookie_Jar::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php on line 89

Deprecated: Return type of Requests_Cookie_Jar::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php on line 102

Deprecated: Return type of Requests_Cookie_Jar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php on line 111

Deprecated: Return type of Requests_Utility_CaseInsensitiveDictionary::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Utility/CaseInsensitiveDictionary.php on line 40

Deprecated: Return type of Requests_Utility_CaseInsensitiveDictionary::offsetGet($key) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Utility/CaseInsensitiveDictionary.php on line 51

Deprecated: Return type of Requests_Utility_CaseInsensitiveDictionary::offsetSet($key, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Utility/CaseInsensitiveDictionary.php on line 68

Deprecated: Return type of Requests_Utility_CaseInsensitiveDictionary::offsetUnset($key) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Utility/CaseInsensitiveDictionary.php on line 82

Deprecated: Return type of Requests_Utility_CaseInsensitiveDictionary::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Utility/CaseInsensitiveDictionary.php on line 91

Warning: Cannot modify header information - headers already sent by (output started at /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php:15) in /usr/www/users/deniserb/public/bsh/wp-admin/includes/misc.php on line 1416

Warning: Cannot modify header information - headers already sent by (output started at /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php:15) in /usr/www/users/deniserb/public/bsh/wp-includes/functions.php on line 6865

Warning: Cannot modify header information - headers already sent by (output started at /usr/www/users/deniserb/public/bsh/wp-includes/Requests/Cookie/Jar.php:15) in /usr/www/users/deniserb/public/bsh/wp-admin/admin-header.php on line 9

jar.php appears regularly any ideas.

Aidan

Tel +49 162 607 9496Email ***@***.*** 

    On Monday, 7 November 2022 at 14:09:33 CET, Basti ***@***.***> wrote:  
 
 



I am getting lots of internal server errors which I am unable to find the source of.


This has for sure nothing to do with the theme. Try to install a copy to another server.

**Comment by aidanrmwatt** [comment]
Did a full back up of mywebsitte using server stored back up. Everything appeared as usual. Wordpress just updated to 6.1 and it broke!! So guessing the Bootscore theme is not seeing eye to eye with WP.
Regards

Aidan

Tel +49 162 607 9496Email ***@***.*** 

    On Monday, 7 November 2022 at 14:09:33 CET, Basti ***@***.***> wrote:  
 
 



I am getting lots of internal server errors which I am unable to find the source of.


This has for sure nothing to do with the theme. Try to install a copy to another server.

**Reply by crftwrk** [reply]
bootScore is just a theme and does nothing in WP core. As mentioned above, you can create an admin account and send it to store(at)bootscore.me. Then we can have a look to it.
